src/elm/ratio.coffee

```diff
@@ -36,5 +36,9 @@ module.exports.Ratio = class Ratio extends Expression
       divided_other = other.numerator.dividedBy(other.denominator)
       divided_this.equals(divided_other)
 
+  equivalent: (other) ->
+    if other instanceof Ratio
+      @equals(other)
```

Comment:
This returns null if `other` is not a Ratio correct, was that what was decided?

Reply:
Also, if this just calls the equals class in Ratio, that also does a check for `other instanceof Ratio` so I don't think you need a duplicate check here, unless we want to return a different "falsey" value for equivalence.